ZelBack/src/services/appsService.js

```diff
@@ -4739,16 +4774,40 @@ async function verifyAppMessageSignature(type, version, appSpec, timestamp, sign
  * @param {number} timestamp Time stamp.
  * @param {string} signature Signature.
  * @param {string} appOwner App owner.
+ * @param {number} daemonHeight Daemon height.
  * @returns {boolean} True if no errors are thrown.
  */
-async function verifyAppMessageUpdateSignature(type, version, appSpec, timestamp, signature, appOwner) {
+async function verifyAppMessageUpdateSignature(type, version, appSpec, timestamp, signature, appOwner, daemonHeight) {
   if (!appSpec || typeof appSpec !== 'object' || Array.isArray(appSpec) || typeof timestamp !== 'number' || typeof signature !== 'string' || typeof version !== 'number' || typeof type !== 'string') {
     throw new Error('Invalid Flux App message specifications');
   }
+  let marketplaceApp = false;
+  let fluxSupportTeamFluxID = null;
   const messageToVerify = type + version + JSON.stringify(appSpec) + timestamp;
-  let isValidSignature = verificationHelper.verifyMessage(messageToVerify, appOwner, signature); // only btc
-  if (timestamp > 1688947200000) { // remove after this time passed
-    isValidSignature = signatureVerifier.verifySignature(messageToVerify, appOwner, signature); // btc, eth
+  let isValidSignature = signatureVerifier.verifySignature(messageToVerify, appOwner, signature); // btc, eth
+  if (isValidSignature !== true) {
+    const teamSupportAddresses = getChainTeamSupportddressUpdates();
+    if (teamSupportAddresses.length > 0) {
+      const intervals = teamSupportAddresses.filter((interval) => interval.height <= daemonHeight);
+      if (intervals) {
+        const addressInfo = intervals[intervals.length - 1];
+        if (addressInfo.height >= daemonHeight) {
+          fluxSupportTeamFluxID = addressInfo.address;
+          const numbersOnAppName = appSpec.name.match(/\d+/g);
```

Comment:
Won't this return null on no match so it will raise below when we try to iterate?

Reply:
Can you check if latest commit fixes it? :)

Reply:
Looks good